components/SearchBar/styles.ts

```diff
@@ -1,15 +1,31 @@
 import styled from 'styled-components';
 
 export const SearchBarContainer = styled.div`
-  background-color: #f7f7f7;
-  border: none;
-  border-radius: 8px;
-  width: 30%;
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  align-items: center;
+  position: relative;
+`;
+
+export const IconWrapper = styled.div`
+  position: absolute;
+  left: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  pointer-events: none; /* Prevent the icon from blocking input clicks */
 `;
 
 export const SearchBarInput = styled.input`
-  padding: 8px;
+  padding: 8px 8px 8px 32px;
   border: none;
+  border-radius: 16px;
   background-color: #f7f7f7;
   width: 100%;
+  color: #888;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
```

Comment:
note: in the future, if we want to use the global styles here, we can define SearchBarInput as a `styled(p)`, or whatever the text style is (instead of styled.input), so that we don't have to copy the text styling here. 

Then, in order to use SearchBarInput as an input tag, whenever you use it as a component, you can add the `as` prop, like so: 
```
<SearchBarInput
       as="input"
        type="search"
        placeholder="Search Plant"
        value={searchTerm}
        onChange={e => setSearchTerm(e.target.value)}
      />
```